### ISSUE TITLE
Add martingale grid to buydip strategy

### ISSUE THREAD

**Issue by bitcoiniodev** [andresilvasantos/bitprophet#43]
It would have a great benefit if we could set an average down orders in case of price moves down after the first buy order is filled. In that case we can lower profit price even with large moves down. This can be implemented as an addition to the stoploss method

**Comment by andresilvasantos** [comment]
@Mikepuz yes, I have implemented this manually for my own strategies. To have this easily available is a nice addition.
